collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1506,7 +1508,11 @@ private void moveScreen(Direction direction) {
 
                 formEntryViewModel.moveBackward();
                 formIndexAnimationHandler.handle(formController.getFormIndex());
-                break;
+                return true;
+
+            default:
+                // Java switch statements aren't smart enough to know there are only two possibilities
```

Comment:
Maybe a reason to go with if/else?

Reply:
Yeah, you're right. I just miss `when` so much. It feels like all recent signs (Android tools changes, me making mistakes) are pointing to avoid `switch` altogether. I'll change this.